coverage/branches.csv

```diff
@@ -1,12 +1,13 @@
 AttackEpaminondasMinimax.ts,1
 AwaleRules.ts,2
 AwaleMinimax.ts,2
-AuthenticationService.ts,2
+AuthenticationService.ts,1
 ActivesPartsService.ts,4
 ActivesUsersService.ts,1
 count-down.component.ts,1
 Coord.ts,1
 CoerceoPiecesThreatTilesMinimax.ts,3
+ChatService.ts,1
```

Comment:
what about that :D ?

Reply:
eh, c'est un test non stable car j'ai pas touché à chat service ni à quelque chose de relaté :D